test/support/jest/jest.config.js

```diff
@@ -26,7 +26,8 @@ module.exports = {
     '<rootDir>/test/support/jest/jest.setup.js'
   ],
   'testMatch': [
-    '**/test/spec/**/*.{js,ts}'
+    // '**/test/spec/**/*.{js,ts}'
+    '**/test/spec/idx/recoverPassword.{js,ts}'
```

Comment:
make sure to change this back

Reply:
for sure, did git add all... I should stash those unfinished changes ...

Reply:
removed filter